src/main/resources/static/js/subscription.js

```diff
@@ -466,23 +465,23 @@ jQuery(document).ready(function () {
                     "title": "Action",
                     "data": null,
                     "render": function (data, type, row, meta) {
-                        var unsecureEditAll = isSecured == false;
-                        var securedEditOwn = row.ldapUserName == currentUser && row.ldapUserName != null;
-                        var allUserEditNoOwner = row.ldapUserName.length == 0 && currentUser != undefined;                        
-                        var disableButton = unsecureEditAll || securedEditOwn || allUserEditNoOwner;
-                        
-                        var disablingText = "";                        
-                        if(disableButton == false){                        	
-                        	disablingText = " disabled";
-                        } 
-                        return '<button id="view-' + data.subscriptionName + '" class="btn btn-sm btn-success view_record table-btn">View</button> '
-                            + '<button id="edit-' + data.subscriptionName + '" class="btn btn-sm btn-primary edit_record table-btn"' + disablingText + '>Edit</button> '
-                            + '<button id="delete-' + data.subscriptionName + '" class="btn btn-sm btn-danger delete_record table-btn"' + disablingText + '>Delete</button>';
+                        subscriptionOwner = row.ldapUserName
```

Comment:
Is this still causing a problem or is it fine? Tobias reported the following in browser logs when running without ldap:
TypeError: row.ldapUserName is undefined